### PR TITLE
Add entrypoint

### PR DESCRIPTION
**Pull request by klssmith**
https://trello.com/c/p34EuB5T/708-make-how-we-use-docker-entrypoints-and-commands-consistent-across-our-apps

This adds an entrypoint, since we have decided to go for the consistent approach of using Docker entrypoints across our apps.

---

🚨⚠️ This will be deployed automatically all the way to production when you click merge ⚠️🚨

For more information, including how to check this deployment on preview or staging first before it goes to production, see our [team wiki section on deployment](https://github.com/alphagov/notifications-manuals/wiki/Merging-and-deploying#deployment)

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [ "$1" == "web" ]
+then
+  gunicorn --error-logfile - -c /home/vcap/app/gunicorn_config.py application
```
Comment (corlettb) [inline entrypoint.sh:5]: This should be started with exec

File: entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+elif [ "$1" == "web-local" ]
+then
+  flask run --host 0.0.0.0 --port $PORT
```
Comment (corlettb) [inline entrypoint.sh:9]: This should be started with exec

**Review by corlettb — COMMENTED** [review]
I know, as it stands, the current container definition will fall into your "custom command" logic.

I'm presuming you intend to update the terraform to pass in "web" instead after this is merged?

File: entrypoint.sh (hunk 1 above)
Comment (klssmith) [reply entrypoint.sh]: Thanks, this is now updated (and the other place too)

**Comment by klssmith** [comment]
I've got this PR to start using the entrypoint once this and the document-download-api PRs are merged https://github.com/alphagov/notifications-aws/pull/2199

**Review by corlettb — APPROVED** [review]

**Pull request merged**